yarn.lock

```diff
@@ -17341,6 +17341,11 @@ typescript-plugin-css-modules@^1.3.1:
     reserved-words "^0.1.2"
     sass "^1.22.4"
 
+typescript@3.6.2:
+  version "3.6.2"
+  resolved "https://registry.yarnpkg.com/typescript/-/typescript-3.6.2.tgz#105b0f1934119dde543ac8eb71af3a91009efe54"
+  integrity sha512-lmQ4L+J6mnu3xweP8+rOrUwzmN+MRAj7TgtJtDaXE5PMyX2kCrklhg3rvOsOIfNeAWMQWO2F1GPc1kMD2vLAfw==
+
 typescript@^3.6.3:
```

Comment:
`typescript@^3.6.3` didn't get removed. `yarn why typescript` indicates that you need to also change the version in `@thumbtack/thumbprint-react#typescript@3.6.3`.